content/docs/operating/configuration.md

```diff
@@ -399,6 +403,65 @@ region: <string>
 [ port: <int> | default = 80 ]
 ```
 
+### `<openstack_sd_config>`
+
+OpenStack SD configurations allow retrieving scrape targets from OpenStack Nova
+instances.
+
+The following meta labels are available on targets during [relabeling](#relabel_config):
+
+* `__meta_openstack_instance_id`: the OpenStack instance ID
+* `__meta_openstack_instance_name`: the OpenStack instance name
+* `__meta_openstack_instance_status`: the status of the OpenStack instance
+* `__meta_openstack_instance_flavor`: the flavor of the OpenStack instance
+* `__meta_openstack_public_ip`: the public IP of the OpenStack instance
+* `__meta_openstack_private_ip`: the private IP of the OpenStack instance
+* `__meta_openstack_tag_<tagkey>`: each tag value of the instance
+
+See below for the configuration options for OpenStack discovery:
+
+```
+# The information to access the OpenStack API.
+
+# The OpenStack Region.
+region: <string>
+
+# IdentityEndpoint specifies the HTTP endpoint that is required to work with
+# the Identity API of the appropriate version. While it's ultimately needed by
+# all of the identity services, it will often be populated by a provider-level
+# function.
+[ identity_endpoint: <string> ]
+
+# Username is required if using Identity V2 API. Consult with your provider's
+# control panel to discover your account's username. In Identity V3, either
+# UserID or a combination of Username and DomainID or DomainName are needed.
+[ username: <string> ]
+[ userid: <string> ]
+
+# Password for the Identity V2 and V3 APIs. Consult with your provider's
+# control panel to discover your account's preferred method of authentication.
+[ password: <string> ]
+
+# At most one of DomainID and DomainName must be provided if using Username
```

Comment:
I'd spell the field names the same way in the comments as they are named in the YML below (snake_case, not camelCase). Same for the field names below in further sections.